src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
+  </data>
+  <data name="ParenthesisMismatch" xml:space="preserve">
+    <value>The equation is missing an opening paranthesis.</value>
+    <comment>Error that occurs during graping when the equation is missing a (</comment>
+  </data>
+  <data name="UnmatchedParenthesis" xml:space="preserve">
+    <value>The equation is missing a closing paranthesis.</value>
+    <comment>Error that occurs during graping when the equation is missing a )</comment>
+  </data>
+  <data name="TooManyDecimalPoints" xml:space="preserve">
+    <value>Too many decimal points in a number.</value>
+    <comment>Error that occurs during graping when a number has too many decimals. Ex: 1.2.3</comment>
+  </data>
+  <data name="DecimalPointWithoutDigits" xml:space="preserve">
+    <value>Decimal point is missing digits.</value>
+    <comment></comment>
+  </data>
+  <data name="UnexpectedEndOfExpression" xml:space="preserve">
+    <value>Unexpected end of expression.</value>
+    <comment>Error that occurs during graping when the expression ends unexpectedly. Ex: 3-4*</comment>
+  </data>
+  <data name="UnexpectedToken" xml:space="preserve">
+    <value>Unexpected token in the expression.</value>
+    <comment>Error that occurs during graping when there is an unexpected token.</comment>
+  </data>
+  <data name="InvalidToken" xml:space="preserve">
+    <value>Invalid token in the expression.</value>
+    <comment>Error that occurs during graping when there is an invalid token.</comment>
+  </data>
+  <data name="TooManyEquals" xml:space="preserve">
+    <value>Too many equal signs.</value>
+    <comment>Error that occurs during graping when there are too many equals.</comment>
+  </data>
+  <data name="EqualWithoutGraphVariable" xml:space="preserve">
+    <value>Missing x or y variable.</value>
+    <comment>Error that occurs during graping when the equation is missing x or y.</comment>
+  </data>
+  <data name="InvalidEquationSyntax" xml:space="preserve">
+    <value>Invalid equation syntax.</value>
+    <comment>Error that occurs during graping when an invalid syntax is used.</comment>
+    </data>
+    <data name="EmptyExpression" xml:space="preserve">
+    <value>The expression is empty.</value>
+    <comment>Error that occurs during graping when the expression is empty</comment>
+  </data>
+    <data name="EqualWithoutEquation" xml:space="preserve">
+    <value>Equal was used without an equation.</value>
+    <comment>Error that occurs during graping when equal is used without an equation. Ex: sin(x=y)</comment>
+  </data>
+  <data name="ExpectParenthesisAfterFunctionName" xml:space="preserve">
+    <value>Paranthesis missing after function name.</value>
+    <comment>Error that occurs during graping when paranthesis are missing after a function.</comment>
+    </data>
+  <data name="IncorrectNumParameter" xml:space="preserve">
+    <value>A function has the incorrect number of parameters.</value>
```

Comment:
We should reconcile how to refer to part of a function, since we are calling the entire thing a function above.

Reply:
Good point, maybe "Mathematical operation"?